components/Footer/Footer.module.scss

```diff
@@ -211,14 +211,17 @@
     }
 
     @include media-query(menu-break) {
-        flex-direction: row;
+        display: grid;
+        grid-template-columns: 2fr 1fr;
+        align-items: baseline;
         padding: $grid * 3 0 $grid $grid * 3;
```

Comment:
If we want the social media icons to align with our office addresses, we can add equal padding on both left and right sides of this container - `padding: $grid * 3 $grid * 3 $grid $grid * 3;` - as well as removing the transform on the .socialMediaIcons class

Reply:
done this too - don't think this one applies for torchbox.com as it already seems to be aligned